### PR TITLE
Companion for paritytech/substrate#8525 (Council Account)

### PR DESCRIPTION
**Pull request by shawntabrizi**
This is a companion for paritytech/substrate#8525 which enables the council to dispatch calls as a specially derived account id.

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -470,6 +470,7 @@ parameter_types! {
 	pub const CouncilMotionDuration: BlockNumber = 3 * DAYS;
 	pub const CouncilMaxProposals: u32 = 100;
 	pub const CouncilMaxMembers: u32 = 100;
+	pub const CouncilModuleId: ModuleId = ModuleId(*b"py/counc");
```
Comment (shawntabrizi) [inline runtime/kusama/src/lib.rs:473]:
```suggestion
	pub const CouncilModuleId: ModuleId = ModuleId(*b"ku/counc");
```

File: runtime/kusama/src/lib.rs [hunk 2]
```diff
@@ -522,6 +524,7 @@ parameter_types! {
 	pub const TechnicalMotionDuration: BlockNumber = 3 * DAYS;
 	pub const TechnicalMaxProposals: u32 = 100;
 	pub const TechnicalMaxMembers: u32 = 100;
+	pub const TechnicalModuleId: ModuleId = ModuleId(*b"py/techn");
```
Comment (shawntabrizi) [inline runtime/kusama/src/lib.rs:527]:
```suggestion
	pub const TechnicalModuleId: ModuleId = ModuleId(*b"ku/techn");
```

File: runtime/polkadot/src/lib.rs [hunk 3]
```diff
@@ -569,6 +571,7 @@ parameter_types! {
 	pub const TechnicalMotionDuration: BlockNumber = 7 * DAYS;
 	pub const TechnicalMaxProposals: u32 = 100;
 	pub const TechnicalMaxMembers: u32 = 100;
+	pub const TechnicalModuleId: ModuleId = ModuleId(*b"py/techn");
```
Comment (shawntabrizi) [inline runtime/polkadot/src/lib.rs:574]:
```suggestion
	pub const TechnicalModuleId: ModuleId = ModuleId(*b"pd/techn");
```

**Comment by shawntabrizi** [comment]
not needed anymore

**Review by kianenigma — APPROVED** [review]

**Comment by JoshOrndorff** [comment]
Silly bot.

**Comment by kianenigma** [comment]
bad bot, bad bot.

**Comment by shawntabrizi** [comment]
stale

**Pull request closed without merging**